Libs/GObject/Classes/Object.cs

```diff
@@ -261,19 +262,22 @@ protected internal void OnPropertyChanged([CallerMemberName] string? propertyNam
         // This function returns the proxy object to the provided handle
         // if it already exists, otherwise creates a new wrapper object
         // and returns it.
-        public static T WrapPointerAs<T>(IntPtr handle)
+        protected static T WrapPointerAs<T>(IntPtr handle)
+            where T : Object
         {
             if (TryWrapPointerAs<T>(handle, out T obj))
                 return obj;
 
-            throw new Exception($"Failed to wrap handle as type <{typeof(T).FullName}>");
+            throw new Exception("Could not wrap handle");
```

Comment:
I think the fully detailed exception is better...

Reply:
Yeah, this is another regression from the merge. Fixed in latest commit